src/main.ts

```diff
@@ -1,20 +1,23 @@
 import {getInput} from '@actions/core'
 import {getOctokit} from '@actions/github'
 
-async function run(): Promise<void> {
+const run = async (): Promise<void> => {
   try {
     const token: string = getInput('token')
-    const login: string = getInput('username')
+    const username: string = getInput('username')
+
+    console.log(`Following user: ${username}`)
 
     const result = await getOctokit(token).rest.users.follow({
-      username: login
+      username
     })
-    console.log('Result : ' + result.status)
+
+    console.info(`Result: ${result.status}`)
   } catch (error: unknown) {
     if (error instanceof Error) {
-      console.log(error.message)
+      console.warn(`Error: ${error.message}`)
```

Comment:
IMHO, `error` should be printed with an error level, unless the result is purely informative.

Reply:
Yes, the level is warning, because although it is an error, it does not affect the execution of the action (considered a success). In that sense, it is indeed informative.